software/Server/Python/src/msxpi-server.py

```diff
@@ -115,164 +115,104 @@ def SPI_MASTER_transfer_byte(byte_out):
     #print "transfer_byte:received",hex(byte_in),":",chr(byte_in)
     return byte_in
 
-def piexchangebyte(checktimeout,mypibyte):
-    time_start = time.time()
-    timeout = False
+def piexchangebyte(byte_out):
     rc = RC_SUCCESS
     
     GPIO.output(rdyPin, GPIO.HIGH)
-    #GPIO.wait_for_edge(csPin, GPIO.FALLING)
-    if (checktimeout):
-        while(GPIO.input(csPin)):
-            if (time.time() - time_start > BYTETRANSFTIMEOUT):
-                timeout = True
-    else:
-        while(GPIO.input(csPin)):
-            ()
+    while(GPIO.input(csPin)):
+        ()
```

Comment:
I think, this is a busy-loop creating empty tuples. is it intentional?

Reply:
No, you right, I should have used pass and fix it on next commit.